frontend/rolecall/src/app/user/user-editor.component.html

```diff
@@ -12,8 +12,12 @@ <h1 class="left-card-title">
             [ngClass]="user.uuid == currentSelectedUser.uuid ? 'selected-user' : ''"
             (click)="setCurrentUser({user : user, fromInputChange : false, shouldSetLastUser : true})">
           <h2 class="user-name">
-            {{ (user.first_name ? user.first_name : " ") + " " + (user.last_name ? user.last_name
-              : " ") }}
+            {{
+              (user.first_name ? user.first_name + " " : "") + 
+              (user.middle_name ? user.middle_name + " " : "") +
+              (user.last_name ? user.last_name : "") + 
+              (user.suffix ? user.suffix : "") 
+            }}
```

Comment:
Missing space before suffix.

Reply:
This is actually a 'feature'. The 'how to write a suffix' manuals are split on how to write John Smith junior. About half say it should be John Smith, Jr and the other half say it should be John Smith Jr. The easiest way to handle this is to give the user full control over the spacing. We discussed this with the Alving Ailey people and they liked this solution.